docs/docs/about/comparison-powershell.md

```diff
@@ -291,12 +291,14 @@ ConvertTo-PnPClientSidePage|PnP.PowerShell|
 ConvertTo-PnPPage|PnP.PowerShell|
 Copy-PnPFile|PnP.PowerShell|[spo file copy](../cmd/spo/file/file-copy.md)
 Copy-PnPFolder|PnP.PowerShell|[spo folder copy](../cmd/spo/folder/folder-copy.md)
+Copy-PnPItemProxy|PnP.PowerShell||PnP.PowerShell|
 Copy-PnPList|PnP.PowerShell|
 Copy-PnPTeamsTeam|PnP.PowerShell|[teams team clone](../cmd/teams/team/team-clone.md)
 Deny-PnPTenantServicePrincipalPermissionRequest|PnP.PowerShell|[spo serviceprincipal permissionrequest deny](../cmd/spo/serviceprincipal/serviceprincipal-permissionrequest-deny.md)
 Disable-PnPFeature|PnP.PowerShell|[spo feature disable](../cmd/spo/feature/feature-disable.md)
 Disable-PnPFlow|PnP.PowerShell|[spo flow disable](../cmd/flow/flow-disable.md)
 Disable-PnPPageScheduling|PnP.PowerShell|
+Disable-PnPPowerShellTelemetry|PnP.PowerShell|
```

Comment:
We have a command for this as well: https://pnp.github.io/cli-microsoft365/about/telemetry/#disable-telemetry

Reply:
good catch.. thanks 👍. Will add 😊

Reply:
ok so I checked that and actually, we have a config setting for it, not a dedicated command. Usually we here we define the general command so I would need to add something like `cli config set` which without additional context what to set may not have any sense for the user 🤔. So probably adding something like `m365 cli config set --key disableTelemetry --value true` would be better but then it will not follow the general rule how we mention commands here 😉.
What is your opinion on that?

Reply:
Yes it's tricky, I think that my preference would go to `m365 cli config set --key disableTelemetry --value true`, what do you think @pnp/cli-for-microsoft-365-maintainers?

Reply:
From Usability point of view, I like @milanholemans suggestion to go with giving the command. But that will disturb the consistency. 

How about giving the command directly `cli config set` and linking it to the Telemetry page which we already have. For example for disabling we will link it to our Telemetry page - https://pnp.github.io/cli-microsoft365/about/telemetry/#disable-telemetry?

Reply:
@arjunumenon this and interesting idea 🤔. We will keep the consistency and provide link to the relevant place at the same time 😊
I like it 👍 
+1 ✔

@pnp/cli-for-microsoft-365-maintainers any other comments before I do the update? 
@milanholemans will @arjunumenon idea work for you ?

Reply:
For me it doesn't really matter much, I can live with all ideas here. However consistency is not really an argument here because we're not doing this anywhere else.

Reply:
yes. that was also what I mean 😊. So that in all places of this table we present the commands in similar way. Till now we don't specify options for any command 👍.

Reply:
@milanholemans any comments on this one before I update this or should I proceed and do as @arjunumenon suggested?

Reply:
I'm fine with anything 🙂